.github/workflows/main.yml

```diff
@@ -62,7 +62,9 @@ jobs:
     steps:
 
     - name: checkout
-      uses: actions/checkout@v1
+      uses: actions/checkout@v2
+      with:
+        path: ibis_bigquery
```

Comment:
Need this in https://github.com/ibis-project/ibis-bigquery/blob/main/.github/workflows/system-tests-pr.yml and https://github.com/ibis-project/ibis-bigquery/blob/main/.github/workflows/system-tests.yml as well, I believe.